user/deployment/npm.md

```diff
@@ -132,3 +132,38 @@ after_deploy:
   - ./after_deploy_1.sh
   - ./after_deploy_2.sh
 ```
+
+### Troubleshooting
+
+Did you expect a job to publish a package but it did not? Here are some common reasons that can happen and how to resolve them.
```

Comment:
As I indicated earlier, the first two items are covered elsewhere. Duplicating information is a maintenance burden which should be avoided unless the value of doing so exceeds it. I don't think it does here.